README.rst

```diff
@@ -9,39 +9,49 @@ and ``with_pattern()`` when ``import \*`` is used:
 
 From there it's a simple thing to parse a string:
 
->>> parse("It's {}, I love it!", "It's spam, I love it!")
-<Result ('spam',) {}>
->>> _[0]
-'spam'
+.. code-block:: pycon
```

Comment:
Language should be "python" or "py", but not "pycon" (AFAIK).

IN ADDITION: 
`README.rst` is generated from `parse.py`.
tox runs doctests based on this information.
See "tox.ini".

Reply:
`pycon` is "python console": https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer